ocp_resources/resource.py

```diff
@@ -25,6 +26,7 @@
     ConnectionResetError: [],
     InternalServerError: ["etcdserver: leader changed"],
     ServerTimeoutError: [],
+    ConflictError: [],
```

Comment:
Any `ConflictError` exception handling will need to be limited to within `ResourceEditor`, it should not be added to the defaults for all resources.

Reply:
Ok, I will update `_retry_cluster_exceptions` to accept `exceptions_dict` as a parameter, to achieve this.